src/components/Dropdown/Dropdown.tsx

```diff
@@ -1,11 +1,5 @@
-import { Icon } from "components/Icon/Icon";
-import React, {
-  Dispatch,
-  SetStateAction,
-  useEffect,
-  useRef,
-  useState,
-} from "react";
+import { DropdownInput } from "components/DropdownInput/DropdownInput";
```

Comment:
I think this might need to be a relative import?

Reply:
included